product-tours/tour-pytorch/Tour-with-PyTorch.ipynb

```diff
@@ -0,0 +1,593 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "QXGSKS1u0n8x",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "# Tour with PyTorch"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Install dependencies"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "Before we start, make sure that you have all dependencies installed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "DO7yawFVxVIt",
+    "tags": [
+     "installation"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "! pip install --quiet torch==1.7.1+cpu torchvision==0.8.2+cpu -f https://download.pytorch.org/whl/torch_stable.html scikit-learn==0.24.1 neptune-client==0.5.0"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "DO7yawFVxVIt",
+    "tags": [
+     "installation",
+     "library_updates"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "! pip install --quiet --upgrade torch torchvision -f https://download.pytorch.org/whl/torch_stable.html scikit-learn neptune-client"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Introduction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "This tour will show you how to start using Neptune and PyTorch together. In the following sections you will learn Neptune's basics with typical deep learning classification task.\n",
+    "\n",
+    "In this tour you will learn:\n",
+    "\n",
+    "* how to set project and create experiment in Neptune,\n",
+    "* how to log model parameters, loss, scores and images to experiment,\n",
+    "* where to explore the results."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Logging PyTorch meta-data to Neptune"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Basic example"
```

Comment:
I think it doesn't make sense to have a 'basic example' here as it is the only example in this notebook (atleast as I was reading it that is what I felt). If we drop it we need to update the headers with the proper number of '#'

Reply:
corrected